tests/remote/test_fennec.py

```diff
@@ -19,18 +19,7 @@
     # Support for API level 15 ended on Aug 29th 2017
     {'application': 'fennec', 'platform': 'android-api-15',
      'branch': 'mozilla-central', 'date': '2017-08-29'},
-    {'application': 'fennec', 'platform': 'android-api-16', 'branch': 'mozilla-central'},
     {'application': 'fennec', 'platform': 'android-x86', 'branch': 'mozilla-central'},
-    {'application': 'fennec', 'platform': 'android-api-16',
-     'branch': 'mozilla-central', 'date': '2017-08-30'},
```

Comment:
Actually please only remove those without a date or build id. As it looks like only the latest links have been removed. Builds from 2017 are still around.